p/api/greader.php

```diff
@@ -668,9 +672,6 @@ private static function streamContents(string $path, string $include_target, int
 		header('Content-Type: application/json; charset=UTF-8');
 
 		switch ($path) {
-			case 'reading-list':
```

Comment:
Why did you remove this section? If anything, it can be moved together with default (see proposal below)

Reply:
By deleting it, it is automatically directed to the default

Reply:
Yes I know, but by removing it, you lose the information about what you are supposed to support

Reply:
Right now it's clear that it's supposed to do that. Losing that clarity is not a good thing. :-)